.sync/Version.njk

```diff
@@ -33,4 +33,4 @@
 {% set mu_devops = "v1.4.2" %}
 
 {# The version of the fedora-35-build container to use. #}
-{% set linux_build_container = "ghcr.io/tianocore/containers/fedora-35-build:2113a0e" %}
+{% set linux_build_container = "ghcr.io/tianocore/containers/fedora-35-build:98e849d" %}

```

Comment:
Consider using build with dotnet to fix tests. It is currently building. Will update with tag when it is finished.

Reply:
5b8a008

https://github.com/tianocore/containers/pkgs/container/containers%2Ffedora-35-build

Reply:
I was waiting on the build at the time. Updated to it in PR commit c5da30d.